stdlib/public/core/Sequence.swift

```diff
@@ -607,11 +607,11 @@ public protocol Sequence {
 
   /// Create a native array buffer containing the elements of `self`,
   /// in the same order.
-  func _copyToContiguousArray() -> ContiguousArray<Element>
+  __consuming func _copyToContiguousArray() -> ContiguousArray<Element>
```

Comment:
The "copy" in this and `_copyContents` contradicts the `__consuming`.

Reply:
We could rename the requirement. It would only be implementable for a move-only type by consuming.